DockerFile

```diff
@@ -0,0 +1,15 @@
+FROM node:8-alpine
+
+WORKDIR /opt/datamanager
+
+RUN apk --no-cache add gcc g++ bash musl-dev make python
```

Comment:
Just for curiosity, why we need gcc, python, etc?

Reply:
Hmmm, good question... I made some tests, and it is not necessary to build an image. I will remove that.